src/main/features/core/tray.js

```diff
@@ -105,12 +106,16 @@ const setContextMenu = (track) => {
     },
     { type: 'separator' },
     {
+      type: 'checkbox',
+      checked: PlaybackAPI.getRating().liked,
       label: TranslationProvider.query('playback-label-thumbs-up'),
-      click: () => Emitter.sendToGooglePlayMusic('playback:thumbsUp'),
+      click: () => Emitter.sendToGooglePlayMusic('playback:toggleThumbsUp'),
```

Comment:
Why did you change this?

Reply:
When the menu item is in the checked state and selected, it becomes unchecked and toggles the thumbs up state.